### ISSUE TITLE
Show the status of the overall system in URL embeds using meta tags

### ISSUE THREAD

**Issue by nathen418** [louislam/uptime-kuma#2134]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

It would be nice if the status that shows up in the overall status div on the status page would be in URL embeds. This is similar to what was asked in #1630. 

#1513 partially solves this issue by including the site title and description, but favicon/logos are not embedded. 

Ideally, this would be the case, along with making the rest of the embed more valuable by adding an embed color, emoji for quick status indication, and a description that shows the current overall status. 

This would also make the link embed more pleasing to the eye using other meta tags.

### ✔️ Solution

I would like to see the embed shown below have its meta tags set according to the overall status of the status page, similar to the information displayed in the `overall-status` div.

**Before:**
![image](https://user-images.githubusercontent.com/23101313/192663588-846e0a5b-b520-4ed9-8903-5618602e7aff.png)
**Sample After:**
![image](https://user-images.githubusercontent.com/23101313/192663348-e546649f-ec89-4f3d-964a-4b31b0f6ead0.png)
or
![image](https://user-images.githubusercontent.com/23101313/192663524-98d3a1e2-5106-4d87-8e8f-d04438add85c.png)

Ideally, the `og:image` tag would be replaced with the sites logo, the `theme-color` tag would reflect the current overall status with either green, yellow or red to indicate at-a-glance system status, and the `og:description` tag would reflect the site's title and also state the overall system status as indicated in the `overall-status` div.

Additionally, emojis could be added by using ✅,⚠️, or ❌ in the `og:description` tag like:
`✅ All systems operaitonal`
`⚠️ Partially degraded Service`
`❌ Danger! All systems down`

Another thought would be to include incident text if one exists as an image that shows in the embed, just like how GitHub pull request links embed an image displaying information about the pull request.
A sample image might look something like this:
![image](https://user-images.githubusercontent.com/23101313/192664634-cfb1d9c2-82c4-4bb8-b5e5-16fd67027889.png)

### ❓ Alternatives

I haven't found any alternatives that would achieve this same effect.

**Comment by CL0Pinette** [comment]
I add this to my to-do.
I'll try to implement it asap !

**Comment by nathen418** [comment]
Just checking back if this is a feature that might be possible to add in a future release? Not nagging, just wondering if this has been seen/considered at all

**Comment by CL0Pinette** [comment]
Yeah, sorry, I didn't have time to work on it to this day. And I don't think I'll be able to work on it until one week or two. But as soon as I can, I will implement it 👍

**Comment by nathen418** [comment]
That's ok. Thanks for the quick response though
